docs/userguide/src/en/bpmn/ch05a-Spring-Boot.adoc

```diff
@@ -783,6 +783,13 @@ flowable.http.connection-request-timeout=5s # Connection Request Timeout for the
 flowable.http.request-retry-limit=3 # Request retry limit for the http client
 flowable.http.disable-cert-verify=false # Whether to disable certificate validation for the http client
 
+# Flowable REST
+flowable.rest.app.corsEnabled=true # Whether to enable CORS requests at all. If false, the other properties have no effect
```

Comment:
I guess the properties should use in kebab case (e.g. `flowable.rest.app.cors-enabled`) instead of camel case, to keep the existing style (both ways should work if I am not mistaken).

Reply:
@tstephen could you change the property name to flowable.rest.app.cors-enabled? Sorry for the hassle :-)